client/src/app/+admin/config/edit-custom-config/edit-custom-config.component.ts

```diff
@@ -49,6 +52,31 @@ export class EditCustomConfigComponent extends FormReactive implements OnInit {
       { value: 4, label: '4' },
       { value: 8, label: '8' }
     ]
+
+    this.transcodingConcurrencyOptions = [
+      { value: 1, label: '1' },
+      { value: 2, label: '2' },
+      { value: 4, label: '4' },
+      { value: 8, label: '8' }
+    ]
+
+    this.transcodingNicenessOptions = [
+      { value: 1, label: '1' },
+      { value: 2, label: '2' },
+      { value: 3, label: '3' },
+      { value: 4, label: '4' },
+      { value: 5, label: '5' },
+      { value: 6, label: '6' },
+      { value: 7, label: '7' },
+      { value: 8, label: '8' },
+      { value: 9, label: '9' },
+      { value: 10, label: '10' },
+      { value: 11, label: '11' },
+      { value: 12, label: '12' },
+      { value: 13, label: '13' },
+      { value: 14, label: '14' },
+      { value: 15, label: '15' },
```

Comment:
This from -20 to 19 (included), but user shouldn't place < 0 (cause if it does server will be stuck) so 1 to 19.
A text box with a form validator requiring a number > 0 and < 20 will be better. Because a 19 elements list is too big.